README.rst

```diff
@@ -1222,69 +1159,36 @@ The output will be like:
         """Code is generated for scenarios which are not bound to any tests."""
 
 
-    @given('I have a custom bar')
+    @given("I have a custom bar")
     def I_have_a_custom_bar():
         """I have a custom bar."""
 
 As as side effect, the tool will validate the files for format errors, also some of the logic bugs, for example the
 ordering of the types of the steps.
 
-Migration of your tests from versions 2.x.x
-------------------------------------------------
 
-In version 3.0.0, the fixtures ``pytestbdd_feature_base_dir`` and ``pytestbdd_strict_gherkin`` have been removed.
+Migration of your tests from versions 3.x.x
```

Comment:
I would leave instructions for migrations for older versions of pytest-bdd.

Reply:
They are quite old by now. Let's not have them in the 4.0.0 branch

Reply:
I also changed readthedocs to use stable as default. I think we could stick to the master as a development branch. readthedocs will pick stable tag names.

Reply:
Since readthedocs now shows master, let's indeed merge changes to master